docs/paper/dev/getting-started/project-setup.md

```diff
@@ -0,0 +1,145 @@
+---
+slug: /dev/project-setup
+---
+
+# Paper Project Setup
+
+As the Paper team primarily uses [IntelliJ IDEA](https://www.jetbrains.com/idea/), this guide will be focused on that IDE. 
+However, the steps below should be applicable to other IDEs as well, with some minor changes.
+
+The paper team uses [Gradle](https://gradle.org/) as its build system, and it's tools are implemented for Gradle. 
+Most of the code below can be altered to work with other build systems, such as Maven, but this guide will only cover Gradle.
+
+Follow the guide [here](https://docs.gradle.org/current/userguide/migrating_from_maven.html) to learn how to migrate from Maven to Gradle.
+
+### Creating a new project
+
+Open your IDE and select the option to create a new project. 
+In Intellij, you will get the option to select the type of project you want to create. 
+Select `Gradle - Kotlin DSL` and click `Create`.
```

Comment:
I'd note you need to select "new project" rather than "empty project" or anything else.